app/controllers/api/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::GreetingsController < ApplicationController
+  def index
+    greeting = Greeting.order(Arel.sql('RANDOM()')).first
+    render json: { greeting: greeting.greeting }
+  end
+end
```

Comment:
When trying to run your app I ran into the following error: 
```
NoMethodError (undefined method `greeting' for nil:NilClass):
  
app/controllers/api/greetings_controller.rb:4:in `index'
```
Please correct this, check the image below for a better understanding:
![Screenshot from 2022-01-18 10-47-17](https://user-images.githubusercontent.com/75709904/149970572-5d92e3ac-774e-47fe-a12a-4e63d3f0f0e6.png)

Reply:
I found the issue. Please run rails:db seed before running the server. 👍